diff -urN biodiverse-r1.0.orig/lib/Biodiverse/Metadata/Indices.pm biodiverse-r1.0/lib/Biodiverse/Metadata/Indices.pm
--- biodiverse-r1.0.orig/lib/Biodiverse/Metadata/Indices.pm 2016-04-08 10:52:20.972635837 +1000
+++ biodiverse-r1.0/lib/Biodiverse/Metadata/Indices.pm  2016-04-06 23:21:18.700062463 +1000
@@ -68,7 +68,7 @@



-Readonly my %dep_types = (
+Readonly my %dep_types => (
     pre_calc_global  => 1,
     pre_calc         => 1,
     post_calc        => 1,
